packages/agent-toolkit/src/mcp/toolkit.ts

```diff
@@ -86,6 +93,40 @@ export class MondayAgentToolkit extends McpServer {
     this.registerSingleTool(manageTool as Tool<any, any>);
   }
 
+  /**
+   * Register widget schemas as a resource
+   */
+  private registerWidgetSchemaResource(): void {
```

Comment:
This shouldn't live here ofc, a general `registerResources` would fit, event platform resources may has it own package.